lib/std/crypto/aes_gcm.zig

```diff
@@ -114,7 +114,6 @@ test "Aes256Gcm - Empty message and no associated data" {
     const ad = "";
     const m = "";
     var c: [m.len]u8 = undefined;
-    var m2: [m.len]u8 = undefined;
```

Comment:
Safe to remove (provided this test is actually run heh).

Reply:
Yes, this is safe to remove. (and yes, the test is actually run)